doc/source/whatsnew/v0.23.0.txt

```diff
@@ -299,6 +299,29 @@ Supplying a ``CategoricalDtype`` will make the categories in each column consist
     df['A'].dtype
     df['B'].dtype
 
+.. _whatsnew_0230.enhancements.str_cat_align:
+
+``Series.str.cat`` has gained the ``align`` kwarg
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, :meth:`Series.str.cat` did not -- in contrast to most of ``pandas`` -- align :class:`Series` on their index before concatenation (see :issue:`18657`).
```

Comment:
make this simpler. This should be just the first section and a previous and new section, see the other whatsnew entires for hints on structure. Add a reference to the docs in text.rst.

Reply:
I'm not sure what "This should be just the first section and a previous and new section" means exactly, but I tried to copy the style of other whatsnew entries. Reference added.